File: src/main/java/com/github/thaynarasilvapinto/SimuladorBanco/repositories/ClienteRepository.java
package com.github.thaynarasilvapinto.SimuladorBanco.repositories;

import com.github.thaynarasilvapinto.SimuladorBanco.domain.Cliente;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ClienteRepository extends JpaRepository<Cliente, Integer> {

}

File: src/main/kotlin/com/github/thaynarasilvapinto/simuladorbanco/repositories/ClienteRepository.kt
package com.github.thaynarasilvapinto.simuladorbanco.repositories

import com.github.thaynarasilvapinto.simuladorbanco.domain.Cliente
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface ClienteRepository : JpaRepository<Cliente, Int> {
    fun findByCpfEquals(CPF: String): Optional<Cliente>
}